libs/template/docs/api/class-directive.md

```diff
@@ -0,0 +1,65 @@
+## ClassDirective
+
+The `rxClass` directive serves a convenient way of using observables for style (class) changes.
+
+Under the hood, it leverages a `RenderStrategy` which in turn takes care of optimizing the change detection
+of your component by passing wanted strategy to `rxClassStrategy` (the default strategy is `'immediate'`). The `ClassDirective` will apply style (class) changes to the template and manage change detection after it got an initial value.
```

Comment:
the part about the default strategy is deprecated since we decided to go with `defaultStrategy`? Though I think having normal as defaultStrategy for setting classes is not the best approach. But we can target this when we refactor the configuration interface for rx-angular.